src/main/java/duke/parser/Parser.java

```diff
@@ -12,6 +12,13 @@
     private static final String ERROR_INVALID_TASK_ID = "The id of the task must be a number. e.g. done 1";
     private static final String ERROR_TOO_MANY_ARGUMENTS = "There are too many arguments for this command.";
 
+    /**
+     * Returns a Command object depending on the command inputted by the User.
+     *
+     * @param fullCommand User input.
+     * @return Command object depending on given input.
+     * @throws DukeException If invalid input.
```

Comment:
Good job writing all the javadocs, I think they are informative, yet kept very concise! For this method's javadoc, I think you can try to further explain what you mean by "invalid input". I feel that it is a little vague, just my two cents!